ONSdigital/dp-frontend-renderer
# Add check to test ShowLink bool value

### What

- Add check to test ShowLink bool value
- Vendor in latest models

### How to review

- Pull https://github.com/ONSdigital/dp-frontend-filter-dataset-controller/pull/163
- Upload a dataset with sparsity such as House prices
- Navigate to geography hierarchy pages and check that "All all" button only shows when options have selected data. It should show if 1 or more has selectable data, but not show if none have selectable data. 

### Who can review

Anyone
